bin/rpi-dump-edid

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+# rpi-dump-edid
+#
+# Copyright (C) 2017 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# This is a simple tool to get the raw EDID of a screen attachd to the RPi.
+# The file is output to the /boot/ partition for easy access.
+#
+# This tool was used for all screen data under kano-settings/res/edid/raw
+# Requires sudo, RPi, and a screen.
+
+
+import os
+import sys
+
+from kano.utils.shell import run_cmd
```

Comment:
Just a question. What should be the general use here for `run_cmd`? `kano.utils.shell` or `kano.utils` ?

Reply:
Looking back at it, the only reason why we have so many `from kano.utils import` statements is because we opted to write a nasty import there for backwards compatibility reasons. 

I think providing the full import path is clearer since you know at a glance where you need to look if you want to see the implementation of that function.

Reply:
Nice, yeah I agree. 👍